examples/compiled/point_invalid_color.vg.json

```diff
@@ -32,14 +32,16 @@
           "x": [
             {
               "test": "!isValid(datum[\"IMDB Rating\"]) || !isFinite(+datum[\"IMDB Rating\"])",
+              "scale": "x",
```

Comment:
FYI, good change as intended in a later fix commit -- invalid values get placed as zero rather than min. 

- value: 0 = min 
- scale: 'x', value: '0' -> encode same output as 0